.github/stale.yml

```diff
@@ -52,7 +52,8 @@ markComment: >
 closeComment: >
   This issue has been automatically closed due to lack of activity. This does
   not mean that we do not value the issue. Feel free to request that it be
-  re-opened if you are going to actively work on it
+  re-opened if you are going to actively work on it, or it is a bug that is
+  reproduceable on the current master branch.
```

Comment:
Bugs aren't closed, and for things that are still reproducible on master that have issues and have gone stale, this will just lead to a loop of people requesting they be reopened, they're reopened, and then they're staled again.

Reply:
so? that's what stale bot is supposed to do.

Reply:
It's not supposed to have pointless churn, which is what this would make.

Reply:
It is not pointless, if a bug stays open, it means somebody recently verified that it is still reproducible.

Reply:
Stale bot won't know this though and it'll close the issue.

Reply:
As I said in my first post, bugs don't get staled.

Reply:
see here for the exempt issue labels https://github.com/CleverRaven/Cataclysm-DDA/blob/b9afd7d1af4272e46820ad64375b3aab8cb03b88/.github/stale.yml#L14-L25